backend/python/app/models/entity.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+from . import db
+from .enum import enum
+
+# common columns and methods across multiple data models can be added via a Mixin class:
+# https://docs.sqlalchemy.org/en/13/orm/extensions/declarative/mixins.html
+
+# see examples of Mixins in current and past Blueprint projects:
+# https://github.com/uwblueprint/dancefest-web/blob/master/db/models.py#L10-L70
+# https://github.com/uwblueprint/plasta/blob/master/backend/app/models/mixins.py#L10-L95
+
+class Entity(db.Model):
+    # define the entities table
+
+    __tablename__ = 'entities'
+
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    string_field = db.Column(db.String, nullable=False)
+    int_field = db.Column(db.Integer)
+    enum_field = db.Column(enum)
+    string_array_field = db.Column(db.ARRAY(db.String))
+    bool_field = db.Column(db.Boolean)
```

Comment:
For consistency with our TypeScript-Postgres and TypeScript-Mongo models let's make these all non-nullable